### DISCUSSION TITLE
matter virtual humidity sensor

### DISCUSSION THREAD

**Discussion by jpadie** [arendst/Tasmota#21303] [Q&A]
hello all
I've created a virtual humidity sensor in the matter config page.  on a conttroller that's already paired with Alexa.  

the humidity sensor does not show up in Alexa (even as a range-controller or similar) and I don't see anything in the logs that might give any insight as to why (e.g. not supported or some other error code). 

any thoughts as to why?

...added some logs

12:39:51.365 MTR: adding endpoint = 6 type:v_humidity name:humidity
12:39:51.412 MTR: =Saved     parameters and configuration
12:40:38.471 CMD: setoption83 1
12:40:38.479 MQT: rab/upstairs/doubleRoom_bedSwitch/stat/RESULT = {"SetOption83":"ON"}
12:42:06.069 MTR: New Commissioning (PASE id=8491) from [10.10.10.52]:5541
12:42:09.466 MQT: rab/upstairs/doubleRoom_bedSwitch/tele/RESULT = {"Matter":{"Commissioning":0}}
12:42:11.110 MTR: New Connection (CASE id=64501) from [10.10.10.52]:5541
12:42:12.013 MTR: =Saved     0 fabric(s) and 0 session(s)
12:42:13.004 MTR: =Saved     1 fabric(s) and 1 session(s)
12:42:13.055 MTR: --- Commissioning complete for Fabric '026931878293FEB2' (Vendor Amazon Alexa) ---
12:42:16.302 MTR: Write_Attr [01]002A/0000 (DefaultOtaProviders) - STATUS: 0x88 
12:42:16.311 MTR: Write_Attr [01]002A/0000 (DefaultOtaProviders) - STATUS: 0x88 
12:42:47.240 MTR: New Connection (CASE id=14299) from [10.10.10.59]:5541
12:42:47.715 MTR: New Connection (CASE id=39829) from [10.10.10.61]:5541
12:42:48.181 MTR: New Connection (CASE id=59723) from [10.10.10.191]:5541
12:42:48.695 MTR: New Connection (CASE id=65061) from [10.10.10.72]:5541
12:42:49.168 MTR: New Connection (CASE id=33838) from [10.10.10.60]:5541
12:42:49.647 MTR: New Connection (CASE id=13195) from [10.10.10.158]:5541
12:42:50.644 MTR: =Saved     1 fabric(s) and 2 session(s)
12:42:51.110 MTR: New Connection (CASE id=30798) from [10.10.10.96]:5541
12:42:51.906 MTR: =Saved     1 fabric(s) and 3 session(s)
12:42:52.694 MTR: =Saved     1 fabric(s) and 4 session(s)
12:42:53.504 MTR: =Saved     1 fabric(s) and 5 session(s)
12:42:54.276 MTR: =Saved     1 fabric(s) and 5 session(s)
12:42:55.137 MTR: =Saved     1 fabric(s) and 5 session(s)
12:42:56.057 MTR: =Saved     1 fabric(s) and 5 session(s)
12:43:04.180 MTR: New Connection (CASE id=3991) from [10.10.10.59]:5541
12:43:04.845 MTR: New Connection (CASE id=63253) from [10.10.10.61]:5541
12:43:06.658 MTR: =Saved     1 fabric(s) and 5 session(s)
12:43:07.839 MTR: =Saved     1 fabric(s) and 5 session(s)
12:43:17.081 MTR: New Connection (CASE id=31124) from [10.10.10.163]:5541
12:43:17.645 MTR: New Connection (CASE id=60221) from [10.10.10.159]:5541
12:43:18.181 MTR: New Connection (CASE id=31214) from [10.10.10.160]:5541

incidentally - i've also noticed that the two tasmotised esp32 boards I have tried this on have gone irretrievably offline about 2 minutes after setting up the virtual humidity sensor.    I will dig these modules out of the wall tomorrow to see whether there is any reported reason of the irretrievable crash (they are not back in factory reset mode, neither is broadcasting an ssid)

**Comment by s-hadinger** [comment]
To my understanding, Alexa does not support humidity sensors as Matter devices. The list of supported devices is here:
https://developer.amazon.com/en-US/docs/alexa/smarthome/matter-support.html



> incidentally - i've also noticed that the two tasmotised esp32 boards I have tried this on have gone irretrievably offline about 2 minutes after setting up the virtual humidity sensor. I will dig these modules out of the wall tomorrow to see whether there is any reported reason of the irretrievable crash (they are not back in factory reset mode, neither is broadcasting an ssid)

I have no idea why this should happen. Even a bootloop should disable Berry after the 4th reboot and make the device reachable again.

**Comment by jpadie** [comment]
noted that it's not on the supported list at the moment, but would you not expect to see a rejection/refusal in the logs?  

power cycle reset is turned off on all my devices to protect against flaky power in SW France.  both devices remained dead until I cycled the breaker for the room.  they went off again after a few minutes.  power cycled again and disabled matter and they have stayed alive.  could have been coincidence - nothing useful in the logs.
